### ISSUE TITLE
Add several live examples

### ISSUE THREAD

**Issue by denar90** [thejameskyle/backbone.service#8]
I propose to add several live examples (jsfiddles or smth) with Marionette and Backbone.

**Comment by ahumphreys87** [comment]
Good idea @denar90 do you use this library? Would you be interested in adding the examples?

**Comment by denar90** [comment]
Wanna use at work and I would love to add some examples 😄
